packages/react-components/react-tags/src/components/TagButton/TagButton.types.ts

```diff
@@ -1,13 +1,12 @@
 import { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 import { TagContextValues, TagProps, TagSlots, TagState } from '../Tag/index';
-import { ARIAButtonSlotProps } from '../../../../react-aria/src/index';
 
 export type TagButtonContextValues = TagContextValues;
 
 export type TagButtonSlots = Omit<TagSlots, 'root' | 'dismissIcon'> & {
   root: NonNullable<Slot<'div'>>;
   dismissButton?: Slot<'button'>;
-  content: NonNullable<ARIAButtonSlotProps<'div'>>;
+  content: Slot<'button'>;
```

Comment:
```suggestion
  content: NonNullable<Slot<'button'>>;
```

This is a required slot right?!